source/explorer/popup_receiver.html

```diff
@@ -1,17 +0,0 @@
-<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML//EN">
-<html>
-<head>
-    <title>Asana Oauth Example: Browser Popup Receiver</title>
-    <script src="asana-explorer.js"></script>
```

Comment:
this wouldn't work, `asana-explorer.js` isn't in this directory - could probably just remove this file?

Reply:
Hmmm. Good point the `index.html` file actually looks in the right directory for it: https://github.com/Asana/developer-docs/blob/master/source/explorer/index.html#L5

Reply:
I'm pretty sure auth is just done via JS within the same window so this is likely unused

Reply:
If this file is being removed should the `REDIRECT_URI ` be updated as well? https://github.com/Asana/api-explorer/blob/6033dd82f04d8f99ddb3225f7694a1a9042d8f60/src/constants.ts#L9-L31

Reply:
Looks like we are passing that `REDIRECT_URI` into the client https://github.com/Asana/api-explorer/blob/6033dd82f04d8f99ddb3225f7694a1a9042d8f60/src/components/explorer.ts#L35. I am guessing it is redirecting to `https://asana.github.io/developer-docs/explorer/popup_receiver.html` (`https://developers.asana.com/explorer/popup_receiver.html`) after the auth. The current behavior takes you to `https://developers.asana.com/explorer`.

Reply:
yeah, took another look and it seems like it's supposed to redirect there? but with the way the script is referenced it should error out. yeah i'll deploy it out and test it that it works